extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -776,36 +814,33 @@ export class AmpLiveList extends AMP.BaseElement {
    * @private
    */
   isValidChild_(child) {
-    return !!child.hasAttribute('id') &&
-        Number(child.getAttribute('data-sort-time')) > 0;
+    return !!child.hasAttribute('id') && child.hasAttribute('data-sort-time') ?
```

Comment:
In some places we just rely on the presence of the `data-sort-time` attribute (e.g. `!sortTime` above), and here we actually check its value.  Should these all be consistent and either (a) always just check the presence, or (b) always check the value?

Reply:
Done.